README.md

```diff
@@ -23,10 +23,44 @@
 
 <p><img src="schedule.png" alt="LTS Schedule"/></p>
 
-The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
+The Release schedule is available also as a [JSON][] file or [ICal][]. There is
 also a live [Google Calendar][] that may be subscribed to.
 
-# LTS Plan
+## Mandate
+
+The Release working group's purpose is:
+
+* Management/execution of the release process for all releases.
+
+Its responsibilities are:
+
+* Define the release process.
+* Define the content of releases.
+* Generate and create releases. 
+* Manage the LTS and Current branches including backporting changes to 
+  these branches.
+* Define the policy for what gets backported to release streams.
+
+The Release working group is structured into teams and membership in
+the working group does not automatically result in membership in these
+teams. These teams are:
+
+* Releasers team
+* LTS team
+* CITGM team
+
+The `releasers` team is entrusted with the secrets and CI access to be able
+build and sign releases. **Additions to the releasers team must be approved
+by the CTC.**
+
+The Long Term Support (LTS) team manages the process/content of LTS releases
+and the required backporting for these releases.
+
+The Canary in the Gold Mine (CITGM) team maintains CITGM as one of
+the key sanity checks for releases. This team maintains the CITGM
+implementation and works to keep CITGM builds running and passing regularly.
```

Comment:
Maybe change `implementation` -> `repository` to make it clear that nodejs/citgm isn't managed by the Release WG.